limitador/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "limitador"
 version = "0.2.0"
-authors = ["David Ortiz <z.david.ortiz@gmail.com>"]
+authors = ["David Ortiz <z.david.ortiz@gmail.com>", "Eguzki Astiz Lezaun <eguzki@redhat.com>", "Alex Snaps <asnaps@redhat.com>"]
 license = "Apache-2.0"
 keywords = ["rate-limiting", "rate", "limiter"]
 categories = ["web-programming"]
 description = "Rate limiting library"
-homepage = "https://github.com/kuadrant/limitador"
+homepage = "https://kuadrant.io"
```

Comment:
not sure about this, since this crate is publish in crates.io

Aligned with the [previous](https://github.com/Kuadrant/limitador/pull/112/files#r958547115) comment, I would leave it as it was, but it is ok as you propose.